Jenkinsfile

```diff
@@ -64,16 +58,7 @@ pipeline {
 		stage('Setup: Build jar files'){
 			steps{
 				script{
-					sh "mvn clean compile assembly:single"
-					sh "git clone https://github.com/reactome/data-release-pipeline"
-					// Temporary approach to setting up post-step QA artifact, which comes from a branch in data-release-pipeline. 
-					// Eventually will be moved to something like the release-common-lib repo.
-					dir("data-release-pipeline"){
-						sh "git checkout feature/post-step-tests-stid-history"
-						dir("ortho-stable-id-history"){
-							sh "mvn clean compile assembly:single"
-						}
-					}
+					utils.buildJarFile()
```

Comment:
Which JAR file does this build?

Reply:
Its a generic command that so far has worked for all release steps (`mvn clean compile build:assembly`)